src/bundle/Resources/config/graphql/Content.types.yaml

```diff
@@ -88,10 +88,10 @@ ContentRelation:
                 type: "String"
             sourceContent:
                 type: "DomainContent"
-                resolve: "@=value.sourceContentInfo"
+                resolve: "@=resolver('DomainContentItem', [{id: value.sourceContentInfo.id}, null])"
```

Comment:
To reduce technological debt, please use `query` instead of `resolver` which is deprecated. Example usage: https://github.com/ibexa/product-catalog/blob/main/src/bundle/Resources/config/graphql/types/Product.types.yaml#L68.